problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -57,7 +57,7 @@
 					$\mat{5\\-5}$ and
 					$\Set*{\vec v\in\R^2 \given
 					\vec v=k\mat{1\\-1}\text{ for some }k\in\R}$
-				\item $\mat{-1 \\ 2 \\ 1}$ and
+				\item $\mat{-1 \\ 2 \\ 1}$ and \\
```

Comment:
Why did you force linebreaks? This is generally a bad idea...

Reply:
The original overlaps with added questions as the text for these are too wide. I tried to use align but it didn't turn out well. Is there a better way to avoid overlapping text?

Reply:
Take out the forced newlines. I will deal with formatting when edits are complete.